### DISCUSSION TITLE
Issue updating Custom Field plugin - Error:could not find driver

### DISCUSSION THREAD

**Discussion by fabio-pif** [kimai/kimai#4516] [General] [answered]
Following the instruction for updating the "Custom Fileds plugins", when i try to ipdate database with
 bin/console kimai:bundle:metafields:install

I got an error message, see below

Starting installation of plugin: MetaFieldsBundle ...
=====================================================

 [ERROR] Failed to install database for bundle MetaFieldsBundle. An exception
               occurred in the driver: could not find driver

I'm running Kimai 2.7.0 on Ubuntu Linux 22.04.3 with php 8.1.27

How can i fix it?
Thx's
Fabio

**Comment by kevinpapst** [comment]
You either upgraded PHP as well or you are using the wrong PHP on the CLI.
Anyway, that error is not related to the bundle, but to your PHP setup.
The error means Kimai cannot use the configured database connection, because the database driver is missing.

You can check the logfile at kimai/var/log/prod.log
You can run `php -v` to check which PHP is used and `php -m` to check which modules are loaded: pdo_mysql, mysqli are important.

**Reply by fabio-pif** [reply, marked as answer]
Thank you, in fact with php -v i see i was running 8.2, after installing php8.2-mysql now is all OK.
Fabio
